src/Forms/Prism.Forms/Prism.Forms.csproj

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Xamarin.Forms" Version="3.6.0.344457" />
+    <PackageReference Include="Xamarin.Forms" Version="4.5.0.657" />
```

Comment:
this is not compatible with MonoAndroid81... please change the TargetFrameworks to:

```xml
<TargetFrameworks>netstandard2.0;MonoAndroid90;MonoAndroid10.0</TargetFrameworks>
```

Reply:
I should add the Xamarin.Forms team released 4.6 today, go ahead and update to 4.6

Reply:
No worries. I’ve been a bit busy with boring, real life stuff for the last few days. But I’ll be back to this over the weekend. Cheers!